explorer/src/components/account/UpgradeableLoaderAccountSection.tsx

```diff
@@ -122,6 +125,38 @@ export function UpgradeableProgramSection({
             {programData.authority !== null ? "Yes" : "No"}
           </td>
         </tr>
+        <tr>
+          <td>Anchor Verified</td>
```

Comment:
In case people don't know what Anchor is, can you linkify "Anchor" and point it to https://anchor.projectserum.com/

Reply:
Or https://project-serum.github.io/anchor/getting-started/verification.html#building ?

Reply:
"Anchor Verifiable Build Status" ?

"verified" might say too much if someone doesn't know what this means.  I could totally write a rug pull with verifiable build

Reply:
That's a bit long for mobile I think, maybe "Program Verification Status"?

Reply:
I think "Build" is the key thing to communicate

Reply:
I also agree that it should not be misconstrued as an "official" stamp of approval on the safety of users of the program. "Anchor Verifiable Build Status" / "Last Anchor Verifiable Build Deployment Slot" would be ideal, but is a bit of a mouthful.

How about "Verifiable Build Status" / "Last Verifiable Build Slot" ? I can try and add a question mark icon next to it, with a pop-over explaining it's a service provided by Anchor, and link to https://project-serum.github.io/anchor/getting-started/verification.html#building ?

Reply:
![image](https://user-images.githubusercontent.com/95379755/151858267-e5a042a1-6362-4d12-9cf7-f4b40320e7f5.png)
How about that?

Reply:
> How about "Verifiable Build Status"
WFM.  WDYT about then switching from "Yes"/"Verified" to the verification method? So we'd have an "Anchor" badge in this case.  Not sure how this will play with the slot though.  Seems like in reality we don't care about `last_verified_slot < last_deployed_slot`.  Verification should be immediately invalidated upon deployment

Reply:
> WDYT about then switching from "Yes"/"Verified" to the verification method?
I was thinking of something in that line, from the perspective of not giving the monopoly to Anchor, which could merely be **one of the** verification providers, but I think that might be a bit premature given that Anchor is the only service that does that (that I know of at least). We could easily make it like you said if/when a new verification service pops up.

> Not sure how this will play with the slot though. Seems like in reality we don't care about last_verified_slot < last_deployed_slot. Verification should be immediately invalidated upon deployment
The use case I see is: if the program was verified before, the user would be able to go to that build's details, find the git repo, and inspect the diff between then and now to make their call. Versus seeing it's never been verified in which case there's no reference for them.

Reply:
> from the perspective of not giving the monopoly to Anchor

Exactly, this would end up a list of verification providers

> the user would be able to go to that build's details, find the git repo

Can they though? There's nothing tying the deployment slot to the repo.  It's pretty trivial to hide a bugdoor in not too complex a diff

Reply:
> Exactly, this would end up a list of verification providers

Would you agree this can be a step taken later then, or you'd want to see it that way from the start?

> There's nothing tying the deployment slot to the repo

The idea is: user visits explorer -> status shows "unverified" -> last verified build slot shows a slot number -> user clicks on the link icon next to the slot -> leads to anchor build page -> page lists the repo **if the user published from a git repo and that repo is public** (meaning it's optional metadata, you can very well push without)
So there is a link _sometimes_

Reply:
That said @jstarry I checked with Armani and as of 2 days ago all builds pushed to the verification server will automatically got through a verification attempt, so won't remain `built` for long, but instead either `verified` or `mismatch`. So that might be a good argument to get rid of that "last deployed slot" row altogether, also making it simpler.

Reply:
Oh great, yeah removing that row sounds good then

Reply:
> if the user published from a git repo and that repo is public (meaning it's optional metadata, you can very well publish without)
So there is a link sometimes.

It doesn't even matter if the repo is public.  If the current deployment isn't verified, you know nothing about the contract.  Nothing's stopping a malicious dev from pushing juicy features to their repo, then deploying a rug pull instead.

I'm fairly adamant that the verified status should be immediately invalidated upon deploy. False security is worse than no security

Reply:
Yes, I just got what you mean. Even if the diff is available it cannot be trusted, the new version could have been deployed from anywhere, with no ties with the initial repo it was published from. Removed.

Reply:
> Would you agree this can be a step taken later then, or you'd want to see it that way from the start?

How about the `Verified` vs `<insert provider>` badge. What do you guys think?

Reply:
My preference would be to just do the verifier work no, so it's less work when another comes along.  Verifier badge could then link out to the verification page or docs on what exactly verification means in that context

Reply:
Ok how about that.
![image](https://user-images.githubusercontent.com/95379755/152060770-718ff541-235a-4c06-9d00-a40c4859ea58.png)
![image](https://user-images.githubusercontent.com/95379755/152060784-bdf4ab00-d096-4caa-a6f8-02a1a10674ef.png)
![image](https://user-images.githubusercontent.com/95379755/152060877-aceaf2c1-a2bb-4931-95c2-ef91c018d946.png)

Reply:
(just removed the link to Anchor doc from the row label, since it's not Anchor-specific now)